waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -26,10 +26,30 @@ import
 logScope:
   topics = "waku rln_relay"
 
-type MerkleNodeResult* = RlnRelayResult[MerkleNode]
-type RateLimitProofResult* = RlnRelayResult[RateLimitProof]
-type SpamHandler* = proc(wakuMessage: WakuMessage): void {.gcsafe, closure, raises: [Defect].}
-type RegistrationHandler* = proc(txHash: string): void {.gcsafe, closure, raises: [Defect].}
+type WakuRlnConfig* = object
+    rlnRelayDynamic*: bool
+    rlnRelayPubsubTopic*: PubsubTopic
+    rlnRelayContentTopic*: ContentTopic
+    rlnRelayMembershipIndex*: uint
+    rlnRelayEthContractAddress*: string
+    rlnRelayEthClientAddress*: string
+    rlnRelayEthAccountPrivateKey*: string
+    rlnRelayEthAccountAddress*: string
+    rlnRelayCredPath*: string
+    rlnRelayCredentialsPassword*: string
+    
+type 
+  MessageValidationResult* {.pure.} = enum
```

Comment:
I see this one is moved from the waku_rln_relay_types module; curious to know why, any style guide?